Jenkinsfile

```diff
@@ -41,6 +41,7 @@ node("postgresql-9.6") {
         ]
       ]) {
         publishPacts(govuk, env.BRANCH_NAME == 'master')
+        govuk.setEnvar("PACT_CONSUMER_VERSION", "branch-${env.BRANCH_NAME}")
```

Comment:
This might sit a little more coherently with the other pact env vars at the top of the file: https://github.com/alphagov/gds-api-adapters/blob/e54f8aa465e1c31e51f073ad719930aca4bf5080/Jenkinsfile#L8-L9